### PR TITLE
[WIP] Fix service manager enabled

### PR DESCRIPTION
**Pull request by imobachgs**
It depends on https://github.com/yast/yast-yast2/pull/743 and https://github.com/yast/yast-yast2/pull/744.

### CODE REVIEW

File: src/modules/services_manager_service.rb [hunk 1]
```diff
@@ -180,6 +180,7 @@ def extract_services
         service_names.zip(ss).each do |name, s|
           sh = services[name] # service hash
           sh[:enabled] = s && s.enabled?
+          sh[:enabled] = s && (s.enabled? || !!(s.socket && s.socket.enabled?))
```
Comment (jreidinger) [inline src/modules/services_manager_service.rb]: line above should be removed.

Comment (jreidinger) [reply src/modules/services_manager_service.rb]: second and bigger problem is https://trello.com/c/fZC7vbpA/108-3-fate319428-mu-display-correct-status-information-on-services-list#comment-5b30e4a75456b7279ca89c4a as we need to differentiate between enabled and Start on Demand and show to user different string. ( that is reason why card was on blocked, as it does not match description of card, but in the end conclusion is to use this special string for socket )

Comment (imobachgs) [reply src/modules/services_manager_service.rb]: We agreed to just fix the "Enable/Disable" column as a MU and improve the text for SP1.

Comment (jreidinger) [reply src/modules/services_manager_service.rb]: but see kobliha answer there. They want new text in GA and even PMs wants it.

**Comment by imobachgs** [comment]
This fix will be implemented as part of https://trello.com/c/fZC7vbpA/108-3-fate319428-mu-display-correct-status-information-on-services-list.

**Comment by imobachgs** [comment]
https://trello.com/c/fZC7vbpA/108-3-fate319428-mu-display-correct-status-information-on-services-list

**Pull request closed without merging**